### DISCUSSION TITLE
A new Data Explorer and Data Views extension

### DISCUSSION THREAD

**Discussion by anuveyatsu** [ckan/ckan#6069] [Show and tell]
We've been working on [Data Explorer and Data Views library built on React JS](https://github.com/datopian/data-explorer). Some examples of running it in a standalone application using CKAN DataStore APIs:

- https://data.nationalgrideso.com/carbon-intensity1/historic-generation-mix/r/historic_gb_generation_mix
- https://www.energidataservice.dk/tso-electricity/electricitysupplierspergridarea

We're now creating a [CKAN extension](https://github.com/datopian/ckanext-dataexplorer-react) so that this library can be installed into CKAN easily. It is still WIP as we need to improve documentation + need to test on v2.9 etc.

**UPDATE**: the [CKAN extension](https://github.com/datopian/ckanext-dataexplorer-react) is now tested for v2.9 👍

**Comment by anuveyatsu** [comment]
Hi @amercader do you think we could work on getting this as a built-in extension for CKAN core? We believe it could become the replacement for Recline as we've worked on having the same features here.

**Comment by ThrawnCA** [comment]
Your plugin could be quite helpful at www.data.qld.gov.au, since we need to replace Recline, but I notice that you have no tags nor releases.
